src/components/Dashboard/KeywordList.js

```diff
@@ -13,7 +13,9 @@ const KeywordList = ({ keywords, setKeywords, isLoading, setIsLoading }) => {
     const [pageNo, setPageNo] = useState(1);
 
     useEffect(() => {
-        fetchKeywords();
+        setTimeout(() => {
+            fetchKeywords();
+        }, 500)
```

Comment:
using timeout is a code smell. Isn't checking if the token exist before calling other API, including the API to fetch keyword should be sufficient?